includes/common/actions/class-emails.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace um\common\actions;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'um\common\actions\Emails' ) ) {
+
+	/**
+	 * Class Emails
+	 *
+	 * @package um\common\actions
+	 */
+	class Emails {
+
+		public function __construct() {
+			add_action( 'um_mas_send_email', array( $this, 'send' ), 10, 3 );
```

Comment:
Maybe `um_send_email_action` or `um_email_sending` will be more clear name for this hook?